src/styles/_catalog.scss

```diff
@@ -1,22 +1,36 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+$card-width: 200px;
 
-  max-width: 944px;
-  margin: 0 auto;
+.catalog {
+  display: grid;
+  justify-items: center;
+  justify-content: center;
+  gap: 48px;
 
   &__card {
-    display: flex;
+    display: grid;
```

Comment:
Do we need to set a grid for the card ?

Reply:
I know we dont, flex is fine but in the task is said "Replace flex with grid"